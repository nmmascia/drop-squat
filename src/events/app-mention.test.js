const appMention = require('./app-mention');

test('should return the correct app mention message', () => {
  expect(appMention.getMessage({ user: 'nick', channel: 'MY_CHANNEL' })).toMatchSnapshot();
});
