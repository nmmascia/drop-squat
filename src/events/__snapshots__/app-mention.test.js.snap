// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the correct app mention message 1`] = `
Object {
  "as_user": true,
  "attachments": Array [
    Object {
      "actions": Array [
        Object {
          "name": "TALLY_WORKOUT",
          "style": "primary",
          "text": "Tally Workout",
          "type": "button",
        },
        Object {
          "name": "GET_WEEKLY_LEADERBOARD",
          "style": "primary",
          "text": "Get Weekly Leaderboard",
          "type": "button",
        },
      ],
      "callback_id": "???",
      "text": "Hello, <@nick>! What would you like to do today?",
      "title": "Squat Baby Squat",
    },
  ],
  "channel": "MY_CHANNEL",
}
`;
